packages/medusa/src/models/order-edit.ts

```diff
@@ -43,8 +43,8 @@ export class OrderEdit extends BaseEntity {
   @Column({ nullable: true })
   internal_note?: string
 
-  @Column()
-  created_by: string // customer or user ID
+  @Column({ nullable: true })
+  created_by: string // customer, user, third party, etc.
```

Comment:
Suggestion: If we dont make this nullable we shouldn't mark it as so like this 😄 
```suggestion
  @Column()
  created_by: string // customer, user, third party, etc.
```